### PR TITLE
add additional output

### PR DESCRIPTION
**Pull request by mcalhoun**
## what
* add the `iam_role` that was created or passed in as an output to the module

## why
* allows the caller to reference the IAM role in other modules

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -13,6 +13,11 @@ output "storage_bucket_arn" {
   description = "Bucket ARN"
 }
 
+output "iam_role" {
+  description = "IAM Role"
```
Comment (aknysh) [inline outputs.tf:17]: maybe add more specific description what is the role for.
"IAM Role" is too generic

**Review by aknysh — COMMENTED** [review]
see comments

File: outputs.tf (hunk 1 above)
Comment (mcalhoun) [reply outputs.tf]: b9f44f1291121dfe95ded8bf1db52a70084fb773

**Comment by mcalhoun** [comment]
@aknysh I updated the description and now ready to go.

**Review by aknysh — APPROVED** [review]

**Pull request merged**